.github/workflows/ci.yml

```
@@ -59,17 +62,17 @@
     with:
       run-unit: true
       run-component: true
-      run-integration: false
+      run-integration: true
     permissions:
       contents: read
-      actions: write  #? For node caching
+      actions: write      #? For node caching
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)